products/todo/README.md

```diff
@@ -0,0 +1,20 @@
+# Basic TODO Application
+
+This is a basic todo Tracker Application
+
+# Problem Statement
+
+An orderly system is required to track things that need to be done and their status
+
+# Functional Requirements
+- Simple Website
```

Comment:
I don't think this is a good rquirement, please change.

Reply:
I disagree, there needs to be an interface for people to interface with the app